src/modules/editor/ScriptEditorImplementation.cpp

```diff
@@ -67,6 +67,7 @@ extern std::set<ScriptEditorImplementation *> g_pScriptEditorWindowList;
 extern KviModule * g_pEditorModulePointer;
 
 static QColor g_clrBackground(0, 0, 0);
+static QColor g_clrBackgroundHighlight(50, 50, 50);
```

Comment:
> This doesn't force them into using a dark theme. The default theme for the script editor is a dark theme, so the default for "Background highlight" should be for a dark theme as well. It doesn't make sense to default it to match a light theme as the user would have had to change it from dark to something else.

No, of course not, it has to work with dark and light or any other combination ppl want.

My main concern here is theres only a very small range of dark/light colors that will work if you choose one color too light or dark depending you either cant see it or you cant read selected text.

So for this we should add some tooltip explaining limitation.



> I thought about doing lighten(##) but...

idk if its worth the extra work to spend more time on this but tooltip should help some.